variables

```diff
@@ -72,6 +74,7 @@ STAGING
 SPLITUSR
 SUSEMIRROR
 SYSAUTHTEST
+TAPSCRIPT
 TEST
 TOGGLEHOME
 TUMBLEWEED
```

Comment:
Nothing wrong per se, but I think `variables` file is no longer used, or is it? @nilxam @lnussel